mautic/mautic
# control unsubscribe header send

| Q  | A
| --- | ---
| Bug fix? |  No
| New feature? |  Yes
| Automated tests included? | No
| Related user documentation PR URL | mautic/documentation#315
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? |  No
| Deprecations? |  No

#### Description:

Add the ability to disable sending of the `List-Unsubscribe` header in mails.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set disable the unsubscribe header in the email configuration section
3. Send out a test email and make sure that it was not sent with the `List-Unsubscribe` header
